ggml/include/ggml.h

```diff
@@ -254,18 +254,11 @@
 
 #define GGML_PAD(x, n) (((x) + (n) - 1) & ~((n) - 1))
 
-#define GGML_ASSERT(x) \
-    do { \
-        if (!(x)) { \
-            fflush(stdout); \
-            fprintf(stderr, "GGML_ASSERT: %s:%d: %s\n", __FILE__, __LINE__, #x); \
-            ggml_print_backtrace(); \
-            abort(); \
-        } \
-    } while (0)
+#define GGML_ABORT(...) ggml_abort(__FILE__, __LINE__, __VA_ARGS__)
```

Comment:
Please describe the reason to modify this part.

Reply:
This follows the refactor of `GGML_ASSERT` and the addition of `GGML_ABORT` in https://github.com/ggerganov/llama.cpp/pull/8698. 
`GGML_ABORT` is ultilized in CANN related code to abort the process with a message.